### DISCUSSION TITLE
User and Password in go-Echarger

### DISCUSSION THREAD

**Discussion by Goethe14** [syssi/homeassistant-goecharger-mqtt#158] [General]
It is worth mentioning that the user and password must not contain any special characters. It took me a long time to solve the puzzle of 
mqtt://username:password@ipaddress:port in the GO-E app.
Please use User+pwd of the Mosquito Broker in your HA.

**Comment by syssi** [comment]
Do you like to contribute a small addition to the docs? Thanks in advance! It's pretty common to lose time here (see https://github.com/syssi/homeassistant-goecharger-mqtt/issues/149).

**Reply by Goethe14** [reply]
Of course, if I´m able to write something in the docs. I think I have no rights.

**Reply by syssi** [reply]
This is a short guide how to contribute on github: https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project

No explicit write permissions required. You can click the pencil at the top right corner of this page f.e. 

https://github.com/syssi/homeassistant-goecharger-mqtt/blob/main/README.md
